CareKitStore/CareKitStore/CoreData/OCKManagedObjectModel.swift

```diff
@@ -678,6 +756,28 @@ private func makeContactEntity() -> NSEntityDescription {
     return contactEntity
 }
 
+private func makeTaskCategoryEntity() -> NSEntityDescription {
+    let taskCategoryEntity = NSEntityDescription()
+    taskCategoryEntity.name = String(describing: OCKCDTaskCategory.self)
+    taskCategoryEntity.managedObjectClassName = String(describing: OCKCDTaskCategory.self)
+
+    let title = NSAttributeDescription()
+    title.name = "title"
+    title.attributeType = .stringAttributeType
+    title.isOptional = true
+
+    let other = NSAttributeDescription()
+    other.name = "otherTaskCategoryInfoDictionary"
+    other.attributeType = .transformableAttributeType
+    other.isOptional = true
+    other.valueTransformerName = secureUnarchiver
```

Comment:
What is this attribute intended to be used for? I don't think I saw an `@NSManaged` variable for it in `OCKCDTaskCategory`.

Reply:
Forgot to remove this one. Removing now...